src/components/Documentation/Markdown/index.tsx

```diff
@@ -50,11 +50,62 @@ const Abbr: React.FC<{ children: [string] }> = ({ children }) => {
   return <Tooltip text={children[0]} />
 }
 
+const Cards: React.FC<{
+  children: Array<object | string>
+}> = ({ children }) => {
+  return (
+    <div className={styles.cards}>
+      {Array.isArray(children)
+        ? children.reduce(
+            (acc: Array<object>, child, i) =>
+              typeof child === 'object'
```

Comment:
what else can it be ? do we ignore it?

Reply:
This filters out the "\n" strings that occur in remark custom component children, as well as anything else that isn't a `card`, as `cards` components should only contain `card` components.

Reply:
may be it is safer to raise an exception?

Reply:
at least if it something except `\n`

Reply:
`\s*`